src/mpl/src/gpu/mpl_gpu_ze.c

```diff
@@ -93,6 +94,9 @@ static int gpu_ze_init_driver()
 
         if (NULL != global_ze_driver_handle) {
             break;
+        } else {
+            MPL_free(global_ze_devices_handle);
+            global_ze_devices_handle = NULL;
```

Comment:
In the `else-` branch, `global_ze_devices_handle == NULL` already, right? What did I miss?

Reply:
🤦 I see.